src/labone/node_info.py

```diff
@@ -72,11 +72,44 @@ class NodeInfo:
     def __init__(self, info: NodeInfoType):
         self._info: NodeInfoType = info
 
+    @classmethod
+    def plain_default_info(cls, path: str | None = None) -> NodeInfoType:
+        """Get default info for a node.
+
+        Returns:
+            Default info for a node.
+        """
+        info: NodeInfoType = {
+            "Description": "",
+            "Properties": "Read, Write, Setting",
+            "Unit": "None",
+        }  # type: ignore[typeddict-item]
+        if path is not None:
+            info["Node"] = path
```

Comment:
question: What use case is there for a node Info without a path? I would not allow NodeInfos without a path ..